src/test/java/com/google/api/codegen/transformer/go/GoGapicSurfaceTransformerTest.java

```diff
@@ -66,7 +66,9 @@ public static void setupClass() {
             locator,
             new String[] {"myproto_gapic.yaml"});
 
-    productConfig = GapicProductConfig.create(model, configProto, TargetLanguage.GO);
+    productConfig =
+        GapicProductConfig.create(
+            model, configProto, "google.example.myproto.v1", TargetLanguage.GO);
```

Comment:
Pass in null here, instead of a value? Maaaaybe we should add an overload for `GapicProductConfig.create` that matches the original signature and passes through null? I don't feel strongly about that though, so happy to leave asis if you prefer.

Reply:
Done.